src/main/java/com/xiaomi/infra/pegasus/rpc/async/TableHandler.java

```diff
@@ -357,25 +356,21 @@ public void run() {
     }
   }
 
-  void call(final ClientRequestRound round, final int tryId) {
+  void call(final ClientRequestRound round) {
     // tableConfig & handle is initialized in constructor, so both shouldn't be null
     final TableConfiguration tableConfig = tableConfig_.get();
     final ReplicaConfiguration handle =
         tableConfig.replicas.get(round.getOperator().get_gpid().get_pidx());
 
     if (handle.primarySession != null) {
-      // if backup request is enabled, schedule to send to secondary
-      if (round.operator.supportBackupRequest() && isBackupRequestEnabled()) {
-        backupCall(round, tryId);
-      }
-
+      interceptorManger.interceptBefore(round, this);
```

Comment:
I think `before`and `after` are enough to indicate the meaning of these functions.  in my memory,  these name are used in java interceptor.